### PR TITLE
Proposed updates for README

### PR DESCRIPTION
**Pull request by jasonblais**
Some minor changes. ~~I'll add some screenshots and run through the Skype for Business Online instructions when setting up the integration.~~ (Added)

Note that the updated server install instructions assume that the plugin has been migrated to use bot accounts. It can of course be adjusted based on what we decide in #1.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,67 @@
...
-5. Copy the "Application ID" in the "Overview" section
-    1. In Mattermost, go to System Console -> Plugins -> Skype for Business
-    2. Fill in the Client ID using the copied "Application ID" and save the settings.
-    3. Set "Is server version?" to false
+    1. Sign in to https://portal.azure.com/#home
+    2. On the left-hand sidebar, select **Azure Active Directory**, then navigate to **App registrations -> New application registration**
+    3. Set the following values:
+       - **Name**: Mattermost Skype for Business Plugin - [your company name]
+       - **Supoorted account types**: Accounts in any organizational directory and personal Microsoft accounts (e.g. Skype, Xbox, Outlook.com)
+       - **Redirect URI (optional)**: Web, then enter "https://your-mattermost-url.com/plugins/skype4business/api/v1/auth_redirect/" replacing `https://your-mattermost-url.com` with your Mattermost URL
```
Comment (kosgrz) [inline README.md]: Could you remove the slash sign at the end of the URI? It should be like this:
```suggestion
       - **Redirect URI (optional)**: Web, then enter "https://your-mattermost-url.com/plugins/skype4business/api/v1/auth_redirect" replacing `https://your-mattermost-url.com` with your Mattermost URL
```

File: README.md [hunk 2]
```diff
@@ -1,59 +1,67 @@
...
-4. Enable the plugin
-    1. Go to System Console -> Plugins -> Management and click "Enable" underneath the Skype For Business plugin
-5. Try it out
-    1. Click the Skype for Business icon in the channel header
-    2. New post will be created with a link to the newly created meeting
-    3. By clicking the "Join meeting" button, you will be redirected to a meet.<YOUR DOMAIN> page, which in turn will open a Skype for Business client installed on your computer / smartphone to join the meeting
+2. Configure the plugin
+    1. Go to **System Console -> Plugins -> Skype for Business**
+    2. Choose ``Server`` as the **Skype for Business Product Type**
+    3. Enter the **Server Domain** of your Skype for Business server instance. For example, ``contoso.com.``
```
Comment (kosgrz) [inline README.md]: As we agreed to save the credentials in the plugin settings, what about adding new lines with the "Bot username" and the "Bot password" fields? Maybe something like this:

4. Enter the ***Bot username*** of your bot user created in the Skype for Business domain. For example, ``bot@contoso.com``
5. Enter the ***Bot password*** of your bot user created in the Skype for Business domain

?

Comment (jasonblais) [reply README.md]: Indeed 👍

**Comment by jasonblais** [comment]
@kosgrz PR updated per your feedback, thanks!

**Pull request merged**